### PR TITLE
Restructured the Getting Started Section.

### PR DESCRIPTION
**Pull request by mahebbar**
* delete content from overview.rst
* create new files under getting started directory
* add quick install
Referece : #212

### CODE REVIEW

File: documentation/source/getting_started/quick_install.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+You can install OpenEBS instantly using the following procedure.
+
+Prerequisite:
+-------------
+You must have Kubernetes installed.
```
Comment (kmova) [inline documentation/source/getting_started/quick_install.rst]: (Let us make prerequisite as two steps )
1. You must have kubernetes installed
2. You must have iscsi initiator enabled on your minion nodes. The instructions to install / setup iscsi initiator can change depending on your host OS
(Follow 2nd point with example of installing open-iscsi on ubuntu)

File: documentation/source/getting_started/quick_install.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+1. Install Open-ISCSI using the following commands at the command prompt:
+
+   **On Ubuntu -**
+
+   * sudo apt-get update 
+   * sudo apt-get install open-iscsi
+
+   **On CoreOS - ??**
```
Comment (kmova) [inline documentation/source/getting_started/quick_install.rst]: remove this line. I will put in an issue(s) to include the steps for other operating systems.

File: documentation/source/getting_started/quick_install.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+2. Run Kubernetes operator using the following commands at the command prompt:
```
Comment (kmova) [inline documentation/source/getting_started/quick_install.rst]: Convert this into new section : 
Setting up OpenEBS (or something that makes sense). 

It should be -- "Run OpenEBS operator..."

File: documentation/source/getting_started/components_architecture.rst [hunk 4]
```diff
@@ -0,0 +1,23 @@
+Components and Architecture
+===========================
+This section includes OpenEBS components and architecture.
+
+Components
+----------
+
+OpenEBS platform contains three major components:
+
+  * Storage PODs or VSMs
```
Comment (AmitKumarDas) [inline documentation/source/getting_started/components_architecture.rst]: We just wanted to say Storage Containers

File: documentation/source/getting_started/components_architecture.rst [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  * An orchestration engine or VSM Scheduler called Maya
```
Comment (AmitKumarDas) [inline documentation/source/getting_started/components_architecture.rst]: We do not want to talk about VSM

File: documentation/source/getting_started/components_architecture.rst [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  * The OpenEBS hosts that provide the data store from either local disks or remote disks
+
+.. image:: /_static/basic.png
+
+Architecture
+-------------
+
+Maya is the orchestration engine that schedules the VSMs among OpenEBS hosts as needed. Maya driver (Docker Volume Driver for Maya) plays an important role in achieving the smooth flow of provisioning of VSMs and attaining the application consistent snapshots. The data is kept in more than one copy among the OpenEBS hosts through a backend network replication, thus achieving the necessary redundancy. VSMs expose the iSCSI interface currently.
```
Comment (AmitKumarDas) [inline documentation/source/getting_started/components_architecture.rst]: We do not have Docker volume driver for Maya

File: documentation/source/getting_started/components_architecture.rst (hunk 4 above)
Comment (mahebbar) [reply documentation/source/getting_started/components_architecture.rst]: Done! Changes will reflect when I push the document to the readthedocs website.

File: documentation/source/getting_started/components_architecture.rst (hunk 5 above)
Comment (mahebbar) [reply documentation/source/getting_started/components_architecture.rst]: Deleted VSM in the first bullet point. Assuming other instances of VSM remain?

Changes will reflect when I push the document to the readthedocs website.

File: documentation/source/getting_started/components_architecture.rst (hunk 6 above)
Comment (mahebbar) [reply documentation/source/getting_started/components_architecture.rst]: Deleted (Docker volume driver for Maya). Changes will reflect when I push the document to the readthedocs website.

Comment (mahebbar) [reply documentation/source/getting_started/components_architecture.rst]: There is "(Refer to the Roadmap: TBD)" under Architecture section. Where should I refer to?

**Comment by mahebbar** [comment]
The path of the images is corrected. For the terminologies I have raised another issue #229 and #230.

**Pull request merged**